packages/base-styles/_z-index.scss

```diff
@@ -41,6 +41,9 @@ $z-layers: (
 	".wp-block-cover__video-background": 0, // Video background inside cover block.
 	".wp-block-template-part__placeholder-preview-filter-input": 1,
 
+	// Navigation menu dropdown.
+	".has-child .wp-block-navigation-link__container": 28,
```

Comment:
What value is the sibling inserter?

Reply:
Things like the block toolbar and the sibling inserter have been moved to "popovers", that is, children of the `block-editor-block-list__insertion-point-popover` element that sits outside the editing canvas and is then overlaid. That element has z-index 28 also:

<img width="1245" alt="Screenshot 2021-03-10 at 09 44 57" src="https://user-images.githubusercontent.com/1204802/110601601-692b2200-8185-11eb-8a68-83eff43a3af3.png">

https://github.com/WordPress/gutenberg/blob/f8c841abc15f10b34123486baa32d4a752bdf460/packages/base-styles/_z-index.scss#L119